BoxContentSDK/BoxContentSDK/Models/BOXModel.m

```diff
@@ -15,6 +15,8 @@ @implementation BOXModel
 - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
 {
     if (self = [super init]) {
+        self.JSONResponse = JSONResponse;
```

Comment:
Minor - naming: JSONData?

In the context of coming from a network request, json response works, but once you're no longer in that state, it's just the JSON data the represents your model.